### PR TITLE
floating rate flip

### PR DESCRIPTION
**Pull request by minesh-s-patel**
TBD

### CODE REVIEW

File: rosetta-source/src/main/rosetta/observable-asset-calculatedrate-func.rosetta [hunk 1]
```diff
@@ -32,7 +32,7 @@ func EvaluateCalculatedRate: <"Evaluate a calculated rate as described in the 20
         results FloatingRateSettingDetails (1..1) <"detailed results of the floating rate calculation.">
 
     // get the FRO
-    alias fro: floatingRateOption
+    alias fro: interestRateIndex
```
Comment (lolabeis) [inline rosetta-source/src/main/rosetta/observable-asset-calculatedrate-func.rosetta:35]: Adjust the alias name and the comment referring to "FRO"

File: rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta [hunk 2]
```diff
@@ -17,19 +17,19 @@ func IndexValueObservation: <"Retrieve the values of the supplied index on the s
     // data provider - implementation provides observed value from data source
     inputs:
         observationDate date (1..1)
-        floatingRateOption FloatingRateIndex (1..1)
+        interestRateIndex InterestRateIndex (1..1)
```
Comment (lolabeis) [inline rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta:20]: Attributes in this `fro` namespace should be restricted to `FloatingRateIndex` only

File: rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta [hunk 3]
```diff
@@ -17,19 +17,19 @@ func IndexValueObservation: <"Retrieve the values of the supplied index on the s
...
     output:
         observedValue number (1..1)
 
 func IndexValueObservationMultiple: <"Retrieve the values of the supplied index on the specified observation dates.">
     inputs:
         observationDate date (0..*)
-        floatingRateOption FloatingRateIndex (1..1)
+        interestRateIndex InterestRateIndex (1..1)
```
Comment (lolabeis) [inline rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta:27]: Ditto

File: rosetta-source/src/main/rosetta/observable-asset-calculatedrate-func.rosetta (hunk 1 above)
Comment (Oblongs) [reply rosetta-source/src/main/rosetta/observable-asset-calculatedrate-func.rosetta:35]: updated to `alias rate: interestRateIndex` 
references to `fro` in the function updated to `rate`

File: rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta (hunk 2 above)
Comment (Oblongs) [reply rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta]: Moved to `cdm.observable.asset.func`

**Comment by PayalKhanna** [comment]
@Oblongs the expectations are updated on this can you please get it removed

File: rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta (hunk 3 above)
Comment (lolabeis) [reply rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta]: Ditto - Consider moving to `cdm.observable.asset.calculatedrate.func` instead

Comment (Oblongs) [reply rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta]: function moved as suggested

File: rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta (hunk 2 above)
Comment (Oblongs) [reply rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta]: function moved as suggested

**Review by lolabeis — APPROVED** [review]
Previously approved at the [CDM Derivatives Products and Business Events Working Group - October 23rd, 2024](#3168).

Additionally reviewed and approved by maintainer for technical fitness and quality.

**Review by llynhiavu — APPROVED** [review]
Reviewed the main changes

**Pull request merged**